Jellyfin.Plugin.ThePornDB/Helpers/ActorsOverview.cs

```diff
@@ -32,18 +32,26 @@ public static string CustomFormat(JObject actorData)
             };
 
             string overview = Plugin.Instance.Configuration.ActorsOverviewFormat;
-            switch (Plugin.Instance.Configuration.ActorsOverview)
+            
+            switch ((string)actorData["extras"]["gender"])
             {
-                case ActorsOverviewStyle.CustomExtras:
-                    overview = placeholders.Aggregate(Plugin.Instance.Configuration.ActorsOverviewFormat, (current, parameter) => current.Replace(parameter.Key, parameter.Value));
+                case "Female":
+                    overview = Regex.Replace(overview, @"<male>.*?</male>|<trans>.*?</trans>", string.Empty);
                     break;
-                case ActorsOverviewStyle.Default:
-                    overview = (string)actorData["bio"];
+                case "Trans":
+                    overview = Regex.Replace(overview, @"<female>.*?</female>|<male>.*?</male>", string.Empty);
+                    break;
+                case "Male":
+                    overview = Regex.Replace(overview, @"<female>.*?</female>|<trans>.*?</trans>", string.Empty);
                     break;
                 default:
-                    overview = " ";
+                    overview = Regex.Replace(overview, @"<female>.*?</female>|<trans>.*?</trans>|<male>.*?</male>", string.Empty);
```

Comment:
What's point of that `<male>, <female> ets`? it only makes code more complicated

Reply:
And how it works?